pandas/indexes/multi.py

```diff
@@ -856,6 +856,7 @@ def _get_level_values(self, level):
         Parameters
         ----------
         level : int level
+        copy  : bool whether copy of results should be done
```

Comment:
still its not clear why you would actually be making this change, it just adds too much complexity.

show an example of why you think you need it (or simply take it out)